src/codegen/function.rs

```diff
@@ -82,6 +83,12 @@ pub fn generate(
     version_condition(w, env, analysis.version, commented, indent)?;
     not_version_condition(w, analysis.not_version, commented, indent)?;
     doc_hidden(w, analysis.doc_hidden, comment_prefix, indent)?;
+    if !in_trait || only_declaration {
+        doc_alias(w, &analysis.glib_name, comment_prefix, indent)?;
+        if analysis.new_name.is_some() {
+            doc_alias(w, &analysis.name, comment_prefix, indent)?;
```

Comment:
Actually: isn't it still generated doc alias we don't want? Do you have an example now please?

Reply:
https://github.com/fengalin/gtk-rs/blob/309d221a728c5d706cbd553ebe96874a8ec2cba9/glib/src/auto/date_time.rs#L47

Reply:
I only see one `doc(alias = "...")` in this link. Did I miss something?

Reply:
Isn't that what you wanted: https://github.com/gtk-rs/gir/pull/1014#discussion_r536735301?

Reply:
For explicit renames in Gir.toml, you get this: https://github.com/fengalin/gtk-rs/blob/309d221a728c5d706cbd553ebe96874a8ec2cba9/gtk/src/auto/widget.rs#L340

Reply:
At yes, I meant the second, sorry, should have been more explicit. And once again, we fall into the same issue, it generates very common names such `get_name`. :-/

Reply:
Yes, that might be the case with many names. I can remove this, but I thought that would help users figure out changes in names. This is particularly helpful for getters renaming (upcoming).

Reply:
I think it's not a good idea (even less if it's meant to be temporary). What do you think @sdroege ?

Reply:
It's nice that we can serch by the C name, but too bad we don't get results by the actual function name, nor by searching something like "Widget name".

Reply:
Removed